README.md

```diff
@@ -163,6 +172,41 @@ transcripts that are connected to videos in the Wistia studio.
 
 ![disable captions in Wistia](doc/img/wistia_1.png)
 
+### HTML 5
+
+This provider is used to play videos uploaded directly to the Platform.
```

Comment:
I'm not sure about it.
The backend processes the direct URL to the video, it could be stored anywhere including the edx-platform, but not exclusively, am I right?